pages/common/linkchecker.md

```diff
@@ -0,0 +1,20 @@
+# linkchecker
+
+> Command line client to check HTML documents and websites for broken links.
+> More information: <https://linkchecker.github.io/linkchecker/>.
+
+- Find broken links on https://example.com/:
+
+`linkchecker {{https://example.com/}}`
+
+- Also check external URLs:
+
+`linkchecker --check-extern {{https://example.com/}}`
+
+- Ignore URLs that match a specific regex:
+
+`linkchecker --ignore-url {{regex}} {{https://example.com/}}`
+
+- Output results to a file in the csv format:
```

Comment:
Perhaps this will read better?
```suggestion
- Output results to a file in CSV:
```

Reply:
Maybe "Output results to a CSV file"?

Reply:
@Rudloff Let's take `Output results to a CSV file` here.

Reply:
Yeah, that's a better way to word it.